extensions/ql-vscode/src/databases-ui.ts

```diff
@@ -213,113 +330,174 @@ export class DatabaseUI extends DisposableObject {
       showAndLogErrorMessage(e.message);
       return undefined;
     }
-  }
-
-  handleChooseDatabaseInternet = async (): Promise<DatabaseItem | undefined> => {
-    return await promptImportInternetDatabase(this.databaseManager, this.storagePath);
-  }
+  };
+
+  handleChooseDatabaseInternet = async (): Promise<
+    DatabaseItem | undefined
```

Comment:
I've been trying to ignore these reformattings that (to my eye) unnecessarily split lines, but splitting up this type is really weird, given that `Promise<DatabaseItem | undefined>` is kept together on the next function. Is there a setting that's setting a column limit to something significantly less than 80 columns?

Reply:
This is the standard prettier reformatting (recognizing some flags in our settings). And it is aggressive about short lines. I know we're not using prettier in the build. It is currently set to 80 columns. I can make it 100 from now on.